test/TestTaskIncludes.py

```diff
@@ -1,53 +1,50 @@
-        runner = Runner(self.rules, filename, [], [], [])
-        runner.run()
-        self.assertEqual(len(runner.playbooks), 4)
-
-    def test_included_tasks(self):
-        filename = 'test/taskincludes.yml'
-        runner = Runner(self.rules, filename, [], [], [])
-        runner.run()
-        self.assertEqual(len(runner.playbooks), 4)
-
-    def test_include_tasks_2_4_style(self):
-        filename = 'test/taskincludes_2_4_style.yml'
-        runner = Runner(self.rules, filename, [], [], [])
-        runner.run()
-        self.assertEqual(len(runner.playbooks), 4)
-
-    def test_import_tasks_2_4_style(self):
-        filename = 'test/taskimports.yml'
-        runner = Runner(self.rules, filename, [], [], [])
-        runner.run()
-        self.assertEqual(len(runner.playbooks), 4)
-
-    def test_include_tasks_with_block_include(self):
-        filename = 'test/include-in-block.yml'
-        runner = Runner(self.rules, filename, [], [], [])
-        runner.run()
-        self.assertEqual(len(runner.playbooks), 3)
-
-    def test_include_tasks_in_role(self):
-        filename = 'test/include-import-tasks-in-role.yml'
-        runner = Runner(self.rules, filename, [], [], [])
-        runner.run()
-        self.assertEqual(len(runner.playbooks), 4)
+from ansiblelint import Runner
+
+
+def test_block_included_tasks(default_rules_collection):
+    filename = 'test/blockincludes.yml'
+    runner = Runner(default_rules_collection, filename, [], [], [])
+    runner.run()
+    assert len(runner.playbooks) == 4
+
+
+def test_block_included_tasks_with_rescue_and_always(default_rules_collection):
+    filename = 'test/blockincludes2.yml'
+    runner = Runner(default_rules_collection, filename, [], [], [])
+    runner.run()
+    assert len(runner.playbooks) == 4
+
+
+def test_included_tasks(default_rules_collection):
+    filename = 'test/taskincludes.yml'
+    runner = Runner(default_rules_collection, filename, [], [], [])
+    runner.run()
+    assert len(runner.playbooks) == 4
+
+
+def test_include_tasks_2_4_style(default_rules_collection):
+    filename = 'test/taskincludes_2_4_style.yml'
+    runner = Runner(default_rules_collection, filename, [], [], [])
+    runner.run()
+    assert len(runner.playbooks) == 4
+
+
+def test_import_tasks_2_4_style(default_rules_collection):
+    filename = 'test/taskimports.yml'
+    runner = Runner(default_rules_collection, filename, [], [], [])
+    runner.run()
+    assert len(runner.playbooks) == 4
+
+
+def test_include_tasks_with_block_include(default_rules_collection):
+    filename = 'test/include-in-block.yml'
+    runner = Runner(default_rules_collection, filename, [], [], [])
+    runner.run()
+    assert len(runner.playbooks) == 3
+
+
+def test_include_tasks_in_role(default_rules_collection):
+    filename = 'test/include-import-tasks-in-role.yml'
+    runner = Runner(default_rules_collection, filename, [], [], [])
+    runner.run()
+    assert len(runner.playbooks) == 4
```

Comment:
This is essentially a single test with different params:

```suggestion
import pytest


@pytest.mark.parametrize(
    ('filename', 'playbooks_count'),
    (
        pytest.param('blockincludes', 4, id='block included tasks'),
        pytest.param(
            'blockincludes2', 4,
            id='block included tasks with rescue and always',
        ),
        pytest.param('taskincludes', 4, id='included tasks'),
        pytest.param(
            'taskincludes_2_4_style', 4,
            id='include tasks 2.4 style',
        ),
        pytest.param('taskimports', 4, id='import tasks 2 4 style'),
        pytest.param(
            'include-in-block', 3,
            id='include tasks with block include',
        ),
        pytest.param(
            'include-import-tasks-in-role', 4,
            id='include tasks in role',
        ),
    ),
)
def test_block_included_tasks(
        default_rules_collection,
        filename, playbooks_count,
):
    playbook_path = 'test/{filename}.yml'.format(locals())
    runner = Runner(default_rules_collection, playbook_path, [], [], [])

    runner.run()
    assert len(runner.playbooks) == playbooks_count
```

Reply:
Indeed, fixed.

Reply:
FYI it's a good tone to accept suggestions on GitHub UI because it preserves authorship. If you want to squash, you can then do this by preserving `Co-Authored-By` commit trailers